src/components/Button/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import styles from './Button.module.scss';
+import { Loader, LoaderSize } from '../Loader/Loader';
+
+export type ButtonProps = React.PropsWithChildren<{
+  loading?: boolean;
+  children?: React.ReactNode;
+  classname?: string;
+}> &
+  React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const Button: React.FC<ButtonProps> = ({
+  loading,
+  children,
+  className,
+  ...otherActions
+}) => {
+  return (
+    <button
+      className={`${styles.button} ${className} 
+        ${loading || otherActions.disabled ? `${styles.button__disabled}` : ''}
+        `}
+      {...{
+        ...otherActions,
+        ...{ disabled: loading || otherActions.disabled },
+      }}
```

Comment:
зачем так..
можно и красивее вот так:
```suggestion
    {...otherActions},
     disabled={loading || otherActions.disabled},
```

Reply:
Это из дз2 перекочевало, там в следующих задачах покрасивей получалось, а это оптимизировать сначала забыл, потом уже руки не дошли. Благодарю :)